chromadb/db/clickhouse.py

```diff
@@ -263,20 +263,20 @@ def _update(
             update_fields = []
             parameters[f"i{i}"] = ids[i]
             if embeddings is not None:
-                update_fields.append(f"embedding = {{e{i}:Array(Float64)}}")
+                update_fields.append(f"embedding = %(e{i})s")
```

Comment:
Clickhouse throws errors when there are newlines in a parameter when using it's "Server Side Binding" approach. Switching to use [Client Side Binding](https://clickhouse.com/docs/en/integrations/python#client-side-binding) seems to work, so that's what this does.